eng/WpfArcadeSdk/tools/RuntimeFrameworkReference.targets

```diff
@@ -3,6 +3,15 @@
     Explicitly import Microsoft.NETCore.Platforms to prevent using the one from the SDK, which
     usually lags behind the version that WPF gets from CoreFx packages
   -->
+
+  <PropertyGroup>
+    <UpdatedTargetingPackVersion Condition="'$(MicrosoftNETCoreAppRefVersion)'!=''
```

Comment:
As mentioned, all the changes in this file should be reverted.

Reply:
Sorry, conflated this with another change. Reverted.